ios/PolyPodApp/Features/Feature.swift

```diff
@@ -7,6 +7,7 @@ class Feature {
     let author: String?
     let description: String?
     let primaryColor: Color?
+    let thumbnailColor: Color?
```

Comment:
Kotlin does not have that kind of color variable?

Reply:
It's an Android thing I believe.